packages/desktop-client/src/components/schedules/SchedulesTable.js

```diff
@@ -58,15 +61,15 @@ function OverflowMenu({ schedule, status, onAction }) {
             items={[
               status === 'due' && {
                 name: 'post-transaction',
-                text: 'Post transaction'
+                text: t('schedules.postTransaction')
               },
               ...(schedule.completed
-                ? [{ name: 'restart', text: 'Restart' }]
+                ? [{ name: 'restart', text: t('general.restart') }]
                 : [
-                    { name: 'skip', text: 'Skip next date' },
-                    { name: 'complete', text: 'Complete' }
+                    { name: 'skip', text: t('schedules.skipNextDate') },
+                    { name: 'complete', text: t('general.complete') }
```

Comment:
Do you think it would make sense to split “general” up into separate categories for actions and non-actions? I imagine some languages would have different words for “is complete” and “to complete.”

Reply:
In that scenario, the translation key will be "isComplete" and "toComplete"